gazebo_ros_control/CMakeLists.txt

```diff
@@ -31,9 +33,11 @@ catkin_package(
     joint_limits_interface
     urdf
     angles
-  INCLUDE_DIRS include
-  LIBRARIES ${PROJECT_NAME} default_robot_hw_sim
-  DEPENDS gazebo
```

Comment:
given the current state of the issue, I'm ok for not declaring `gazebo` as `DEPENDS` here. Dave, if you could please leave the line commented with a comment pointing to the gazebo issue that should be enough to restore it as soon as we find the way of solving the issue. After the comments, good to merge.